test/extended/olm/olm.go

```diff
@@ -240,25 +240,25 @@ var _ = g.Describe("[sig-operator] an end user can use OLM", func() {
 		oc = exutil.NewCLI("olm-23440")
 
 		buildPruningBaseDir = exutil.FixturePath("testdata", "olm")
-		operatorGroup       = filepath.Join(buildPruningBaseDir, "operatorgroup.yaml")
-		sub                 = filepath.Join(buildPruningBaseDir, "subscription.yaml")
+		ogPath              = filepath.Join(buildPruningBaseDir, "operatorgroup.yaml")
+		subPath             = filepath.Join(buildPruningBaseDir, "subscription.yaml")
+		catalogPath         = filepath.Join(buildPruningBaseDir, "catalogsource.yaml")
 	)
 
-	files := []string{sub}
 	g.It("can subscribe to the operator", func() {
 		g.By("Cluster-admin user subscribe the operator resource")
 
-		// skip test if redhat-operators is not present or disabled
-		ok, err := hasRedHatOperatorsSource(oc)
+		// Configure OperatorGroup before tests
+		og, err := oc.AsAdmin().Run("process").Args("--ignore-unknown-parameters=true", "-f", ogPath, "-p", "NAME=test-operator", fmt.Sprintf("NAMESPACE=%s", oc.Namespace())).OutputToFile("og.json")
+		o.Expect(err).NotTo(o.HaveOccurred())
+		err = oc.AsAdmin().WithoutNamespace().Run("create").Args("-f", og).Execute()
 		o.Expect(err).NotTo(o.HaveOccurred())
-		if !ok {
-			g.Skip("redhat-operators source not found in enabled sources")
```

Comment:
should we be concerned that this check was not working?  Does that mean we're not actually disabling some aspect of the redhat-operators source when marketplace is not enabled?

Reply:
It was testing the OperatorHub API, which still exists as part of a separate problem that I'm currently tracking -- I'll post a PR here when there are more updates.